README.md

```diff
@@ -1,35 +1,62 @@
-Parquet-cpp [![Build Status](https://travis-ci.org/apache/parquet-cpp.svg)](https://travis-ci.org/apache/parquet-cpp)
-===========
-A C++ library to read parquet files.
+## parquet-cpp: a C++ library to read and write the Apache Parquet columnar data format.
+
+[![Build Status](https://travis-ci.org/apache/parquet-cpp.svg)](https://travis-ci.org/apache/parquet-cpp)
 
 ## Third Party Dependencies
+
 - snappy
 - lz4
+- zlib
 - thrift 0.7+ [install instructions](https://thrift.apache.org/docs/install/)
+- googletest 1.7.0 (cannot be installed with package managers)
+
+You can install these dependencies using package manager or using the
+`thirdparty/` scripts in this repository. On Homebrew, you can run:
 
-Many package managers support some or all of these dependencies. E.g.:
 ```shell
-ubuntu$ sudo apt-get install libboost-dev libsnappy-dev liblz4-dev
+ brew install snappy lz4 thrift zlib
 ```
+
+To build the thirdparty libraries in-tree, run:
+
 ```shell
-mac$ brew install snappy lz4 thrift
+./thirdparty/download_thirdparty.sh
+./thirdparty/build_thirdparty.sh
+source thirdparty/set_thirdparty_env.sh
 ```
 
-`setup_build_env.sh` tries to automate setting up a build environment for you
-with third party dependencies.  You use it by running `source
-setup_build_env.sh`.  By default, it will create a build directory `build/`.
-You can override the build directory by setting the BUILD_DIR env variable to
-another location.
+`setup_build_env.sh` sets up a build environment for you with third party
+dependencies.  You use it by running `source setup_build_env.sh`.  By default,
+it will create a build directory `build/`.  You can override the build
+directory by setting the BUILD_DIR env variable to another location.
+
+After building the thirdparty libraries, for future development iteration you
+can set the dependency environment variables (detailed below) by running
+
+`source $BUILD_DIR/thirdparty/set_thirdparty_env.sh`
 
-Also feel free to take a look at our [.travis.yml](.travis.yml) to see how that build env is set up.
+The first time you run `setup_build_env.sh` this is set automatically.
```

Comment:
The environment variables are set automatically the first time you run `setup_build_env.sh`.